examples/prompt_engineering/steps/prompt_testing.py

```diff
@@ -0,0 +1,89 @@
+"""Step for testing and comparing prompts."""
+
+from typing import List
+
+from zenml import step
+from zenml.prompts import Prompt
+
+
+@step
+def compare_prompt_versions(prompt_v1: Prompt, prompt_v2: Prompt) -> dict:
+    """Compare two prompt versions using simple A/B testing logic.
```

Comment:
this step is too simplistic, maybe. at the very least in the documentation you should show something a *bit* more sophisticated, though I think maybe even this comparison feature needs a bit more work.